src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/FixedActiveThreadsExecutor.kt

```diff
@@ -160,7 +160,7 @@ internal class FixedActiveThreadsExecutor(private val nThreads: Int, runnerHash:
             try {
                 runnable.run()
             } catch(e: Throwable) {
-                setResult(iThread, wrapInvalidAccessFromUnnamedModuleExceptionWithDescription(e))
+                setResult(iThread, wrapExceptionIfNecessary(e))
```

Comment:
It's unclear what "necessary" means. In fact, there is only one case when Lincheck handles exception differently -- when an error related to Java modules and missed JVM options has occurred. Let's rename the "wrapExceptionIfNecessary" correspondingly, to make its contract explicit.